R/uvhydrograph-render.R

```diff
@@ -36,10 +36,12 @@ createPrimaryPlot <- function(data, month){
   if(anyDataExist(primaryData)){
 
     primaryInfo <- parseUVSupplemental(data, "primary", primaryData)
+    plotEndDate <- tail(primaryInfo$plotDates,1) + hours(23) + minutes(45)
```

Comment:
@dpattermann-usgs is this the suggestion you have on the premature end of time range issue I'll get to in https://internal.cida.usgs.gov/jira/browse/AQCU-670